lib/config_editor/config_editor_dialog.dart

```diff
@@ -82,7 +82,34 @@ class ConfigEditor extends StatelessWidget {
           ),
         );
         break;
-      case bool:
+      case 'blob':
+      case 'multiline string':
+        final controller = ScrollController();
+        child = SizedBox(
+          width: 200,
+          child: Scrollbar(
```

Comment:
doesn't SingleChildScrollView create a scrollbar automatically on desktop?

Reply:
I don't get a scrollbar without the `Scrollbar` widget. Also, if I use it without the controller it's visible but doesn't work and flutter complains:
```
The Scrollbar's ScrollController has no ScrollPosition attached.
```

Reply:
Ok, nevermind then. Maybe it has something to do with these nested scroll views. :eyes: